### PR TITLE
samples: drivers: spi_flash: Update sample to use alias 

### PR DESCRIPTION
**Pull request by benjaminbjornsson**

### CODE REVIEW

**Review by MaureenHelm — COMMENTED** [review]
Why did you define the alias in overlays instead of in the main board dts?

**Comment by benjaminbjornsson** [comment]
> Why did you define the alias in overlays instead of in the main board dts?

Probably too big of a step without a push... 😄 I'll move it.

File: boards/arm/actinius_icarus_bee/actinius_icarus_bee_common.dts [hunk 1]
```diff
@@ -76,6 +76,7 @@
 		bootloader-led0 = &green_pwm_led;
 		mcuboot-button0 = &button0;
 		mcuboot-led0 = &green_pwm_led;
+		spi-flash = &w25q64;
```
[the remark this answers is not in the document]
Comment (benjaminbjornsson) [reply boards/arm/actinius_icarus_bee/actinius_icarus_bee_common.dts:79]: @fabiobaltieri you're right, I'll fix that, thanks!

**Review by MaureenHelm — DISMISSED** [review]

**Review by fabiobaltieri — DISMISSED** [review]

**Review by jackrosenthal — DISMISSED** [review]

**Review by FRASTM — DISMISSED** [review]

**Review by mmahadevan108 — APPROVED** [review]

**Comment by benjaminbjornsson** [comment]
Cannot re-request review for some reason :)

**Review by fabiobaltieri — APPROVED** [review]

**Comment by fabiobaltieri** [comment]
@anangl can you re-approve?

**Review by anangl — APPROVED** [review]

**Pull request merged**